README.md

```diff
@@ -34,10 +39,109 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
 Here's the quickest way to get going with Rook.
 
-### Linux
+### Kubernetes
+
+This example shows how to build a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
+
+#### Prerequisites
+
+This example requires a running Kubernetes cluster. To make sure you have a Kubernetes cluster that is ready for `rook`, you can [follow these quick instructions](demo/kubernetes/README.md).
+
```

Comment:
@kokhang i would appreciate if you could open any tracking issues that we know of for Kubernetes cluster bootstrapping and being `rook` ready.